layouts/case-studies/immersive.html

```
@@ -0,0 +1,90 @@
+        {{ if $bgImg }}
+        <div class="cs-hero__bg">
+            <img src="{{ $bgImg }}" alt="" />
+        </div>
+        {{ end }}
+        <div class="container">
+            <div class="cs-hero__layout">
+                <div class="cs-hero__inner{{ if $bgImg }} cs-hero__inner--framed{{ end }}">
+                    {{ if .Params.client_logo }}
+                    <div class="cs-hero__lockup">
+                        <div class="cs-hero__lockup-mark cs-hero__lockup-mark--client">
+                            <img src="{{ .Params.client_logo }}" alt="{{ .Params.client | default "" }} logo" />
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)